ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -583,6 +583,8 @@ DFSS:
   - discharge_fan_lost_power_alarm
   - schedule_run_command
   - discharge_fan_run_time_accumulator
+  - discharge_fan_failed_alarm
+  - discharge_fan_run_mode
```

Comment:
really belongs on DFRMM. remove from here and reference DFRMM in your type definition.

Reply:
Removed here and used DFRMM